CHANGELOG.md

```diff
@@ -4,6 +4,64 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# [2.3.0] - 2021-04-19
+
+Props: jrfnl, rebeccahum, kevinfodness, GaryJones.
+
+** There is a breaking change in the ProperEscapingFunction sniff from PR #624 where the $escaping_function property is no longer public anymore due to the formatting change of it. Please remove any usages of the property in custom rulesets.
+
+## Changed
+- [#581](https://github.com/Automattic/VIP-Coding-Standards/pull/581): AlwaysReturnInFilter: flag abstract methods for manual inspection.
+- [#584](https://github.com/Automattic/VIP-Coding-Standards/pull/584): Performance: more selective sniffing for efficiency.
+- [#586](https://github.com/Automattic/VIP-Coding-Standards/pull/586): IncludingNonPHPFile: various improvements.
+- [#587](https://github.com/Automattic/VIP-Coding-Standards/pull/587): LowExpiryCacheTime: new warning added for manual inspection along with various improvements.
+- [#589](https://github.com/Automattic/VIP-Coding-Standards/pull/589): WPQueryParams: inherit from abstract class and flag for 'exclude' parameter.
+- [#592](https://github.com/Automattic/VIP-Coding-Standards/pull/592): DynamicCalls: various improvements.
+- [#595](https://github.com/Automattic/VIP-Coding-Standards/pull/595): Underscorejs: various improvements.
+- [#620](https://github.com/Automattic/VIP-Coding-Standards/pull/620): Ruleset: silence UnusedVariable from VariableAnalysis to reduce noise.
+- [#624](https://github.com/Automattic/VIP-Coding-Standards/pull/624): ProperEscapingFunction: privatize $escaping_functions property along with various improvements.
```

Comment:
This PR added a new check (and error code) to the sniff, so should probably have a separate entry in "Added" and explicitly mention the new functionality (checking for use of `esc_attr()` when escaping something which is not an HTML attribute will now be flagged).

The entry here in "Changed" should probably mention that the sniff now takes more escaping functions into account.

And... I think a separate entry in "Fixed" is also warranted as the PR also fixes various bugs, like where the escaping when HTML attributes used slash escaped quotes in the PHP text string would not correctly be handled/recognized or where the strings using double quotes were not handled correctly etc etc.